CHANGELOG.md

```diff
@@ -21,6 +32,10 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - UI: Accessibility - Make "Tips" heading in Cross Device SMS Sent and Mobile Connected screens a level 3 heading.
 - UI: Accessibility - Change Cross Device Send Link alternate option text to be an ARIA heading level 2
 
+### Added
+
+- Internal: New data field in metadata to track number of takes for each submitted image.
```

Comment:
Looks like this should go under `6.13.0`

Reply:
Good spot!